hello_vsi/hello_vsi.csolution.yml

```diff
@@ -1,13 +1,28 @@
+# Copyright (c) 2024 Arm Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
 solution:
   created-by: uv2csolution@1.2.0
   created-for: CMSIS-Toolbox@2.0.0
-  compiler: AC6
-  cdefault: # use default setup of toolchain specific controls.
+  description: Hello example for the Video Streaming Interface
+
+  packs:
+    - pack: ARM::CMSIS@6.0.0
+    - pack: ARM::CMSIS@6.1.0
+    - pack: ARM::CMSIS-Compiler@2.0.0
```

Comment:
If you take a look in the file `hello_vsi.cproject.yml` you will see that 
- ARM::CMSIS@6.1.0
- ARM::CMSIS-RTX@5.9.0
- ARM::CMSIS-Compiler@2.1.0
are unconditionally selected as well
and 
- ARM::SSE_315_BSP@1.0.0
- ARM::V2M_MPS3_SSE_300_BSP@1.5.0
- ARM::V2M_MSP3_SSE_310_BSP@1.4.0
are selected for the respective target types.
What does that achieve?

Reply:
Has been corrected as follow:

  **packs:**
    - pack: ARM::CMSIS@6.1.0
    - pack: ARM::CMSIS-RTX@5.9.0
    - pack: ARM::CMSIS-Compiler@2.1.0

    - pack: ARM::V2M_MPS3_SSE_300_BSP@1.5.0
      for-context: +Corstone_300
    - pack: ARM::V2M_MPS3_SSE_310_BSP@1.4.0
      for-context: +Corstone_310
    - pack: ARM::SSE_315_BSP@1.0.0
      for-context: +Corstone_315
    - pack: ARM::DMA350@1.0.0
      for-context: +Corstone_315